src/__test__/utils/cellSets/getContainingCellSetsProperties.test.js

```diff
@@ -0,0 +1,13 @@
+import getContainingCellSetsProperties from 'utils/cellSets/getContainingCellSetsProperties';
+import mockCellSets from 'utils/tests/mockStores/cellSets';
+
+describe('get cell class properties tests', () => {
+  const { cellSets } = mockCellSets();
+  it('returns information for a passed cellId', () => {
+    // doing it as a string to make sure it still works (sometimes a string is passed from vitessce)
+    const cellId = '3';
+    const cellSetClassKeys = ['test'];
+    const result = getContainingCellSetsProperties(cellId, cellSetClassKeys, cellSets);
+    expect(result).toMatchSnapshot();
```

Comment:
Can you use a `cellId` that is contained in one of the cell sets? From the snapshot, The returned value doesn't contain anything.